corehq/apps/reminders/management/commands/migrate_to_new_reminders.py

```diff
@@ -502,6 +502,8 @@ def get_rule_recipients(handler):
         return [(CaseScheduleInstanceMixin.RECIPIENT_TYPE_LAST_SUBMITTING_USER, None)]
     elif handler.recipient == RECIPIENT_PARENT_CASE:
         return [(CaseScheduleInstanceMixin.RECIPIENT_TYPE_PARENT_CASE, None)]
+    elif handler.recipient == RECIPIENT_SUBCASE:
```

Comment:
Super picky, but it'd be nice to use "child case" or "subcase" but not both - not sure if one of them is already more common in the code base.

Reply:
agree, actually all these references to subcase are from the old framework (this one in particular is for the migration script) which will get removed soon.

all the references in the new framework use child case and that seems to be more prevalent in our docs

Reply:
great